src/components/App.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <Header :dark-mode="darkMode" />
+  <AppHeader :dark-mode="darkMode" />
   <GoogleForm/>
-  <Footer :darkMode="darkMode" @update:darkMode="toggleDarkMode" />
+  <AppFooter :darkMode="darkMode" @update:darkMode="toggleDarkMode" />
 </template>
 
 <script>
-import Header from './Header'
-import Footer from './Footer'
-import GoogleForm from './GoogleForm'
+import AppHeader from '@/components/AppHeader.vue'
+import AppFooter from '@/components/AppFooter.vue'
```

Comment:
Can we keep the import paths relative? `@` path alias isn't a native node thing. The more vanilla we keep the code the easier it is to test if we wanted to add a test runner or something later.

Reply:
Personally hate relative paths. But ok